mintapi/signIn.py

```diff
@@ -581,13 +645,14 @@ def account_selection_page(driver, intuit_account):
     try:
         select_account = driver.find_element_by_id("ius-mfa-select-account-section")
         if intuit_account is not None:
+   
             account_input = select_account.find_element_by_xpath(
-                "//label/span[text()='{}']/../preceding-sibling::input".format(
+                "//span[text()='{}']/preceding::input[1]".format(
```

Comment:
I'm curious about this section.  Last I tried to work on the Intuit Account section, I wasn't able to get it to work.  This may fix that.  I'd like this to be in a separate branch\PR.  Can you pull this out and submit to the main repo for testing?